src/app/compressed-air-assessment/compressed-air-assessment-results.service.ts

```diff
@@ -521,6 +521,46 @@ export class CompressedAirAssessmentResultsService {
     };
   }
 
+  calculateCompressorSummary(dayTypes: Array<CompressedAirDayType>, compressedAirAssessment: CompressedAirAssessment, settings: Settings): Array<Array<CompressorSummary>> {
+    let compressorSummaries: Array<Array<CompressorSummary>> = new Array<Array<CompressorSummary>>();
+    let compressorInventoryItems: Array<CompressorInventoryItem> = compressedAirAssessment.compressorInventoryItems;
+    dayTypes.forEach(dayType => {
+      let dayTypeCompressorSummaries: Array<CompressorSummary> = new Array<CompressorSummary>();
+      let profileSummary = this.calculateBaselineDayTypeProfileSummary(compressedAirAssessment, dayType, settings);
+      profileSummary.forEach(profile => {
+        let specificPowerAvgLoad = (profile.avgPower/profile.avgAirflow)*100;
+        specificPowerAvgLoad = this.convertUnitsService.roundVal(specificPowerAvgLoad, 4);
+        let compressor: CompressorInventoryItem = compressorInventoryItems.find(compressor => {return compressor.itemId == profile.compressorId})
+
+        let ratedSpecificPower = this.calculateRatedSpecificPower(compressor);
+        let ratedIsentropicEfficiency = this.calculateRatedIsentropicEfficiency(compressor, ratedSpecificPower);
+        let compressorSummary: CompressorSummary = {
+          dayType: dayType,
+          specificPowerAvgLoad: specificPowerAvgLoad,
+          ratedSpecificPower: ratedSpecificPower,
+          ratedIsentropicEfficiency: ratedIsentropicEfficiency
+        }
+        dayTypeCompressorSummaries.push(compressorSummary);
+      });
+      compressorSummaries.push(dayTypeCompressorSummaries);
+    });
+    return compressorSummaries;
+  }
+
+  calculateRatedSpecificPower(compressor: CompressorInventoryItem): number {
+    let ratedSpecificPower: number;
+    ratedSpecificPower = (compressor.nameplateData.totalPackageInputPower / compressor.nameplateData.fullLoadRatedCapacity) * 100;
+    ratedSpecificPower = this.convertUnitsService.roundVal(ratedSpecificPower, 4);
+    return ratedSpecificPower;
+  }
+
+  calculateRatedIsentropicEfficiency (compressor: CompressorInventoryItem, ratedSpecificPower: number): number {
+    let ratedIsentropicEfficiency: number;
+    ratedIsentropicEfficiency = (16.52 *((((compressor.nameplateData.fullLoadOperatingPressure + 14.5)/14.5)^0.2857)-1))/ratedSpecificPower;
```

Comment:
Most important, double check with Alex whether there should be a conversion done in either of these two calculation methods. Pressures look like they go psia/barg, airflow acfm/m3, but don't take my word for it.

Also, if there was a formula for either of these methods it would be nice to add as a comment. Helpful in discerning what the numbers represent.